src/app/shared/components/template/pipes/latex.pipe.ts

```diff
@@ -0,0 +1,59 @@
+import { Pipe, PipeTransform } from "@angular/core";
+import { DomSanitizer, SafeHtml } from "@angular/platform-browser";
+import katex from "katex";
+import { extractMath, Segment } from "extract-math";
+import DOMPurify from "dompurify";
+
+@Pipe({
+  name: "latex",
+})
+export class LatexPipe implements PipeTransform {
+  segments: Segment[] = [];
+
+  constructor(private domSanitizer: DomSanitizer) {}
+
+  transform(latexInput: string): SafeHtml {
```

Comment:
Overall I think this is a nice and tidy implementation, and definitely better than having to import the whole `ng-katex` library. It would be good to have a comment either above the `transform` function or the class name just to give a quick overview of what this does (i.e. checks for latex within text string with various delimeter formats, and renders as html)